### ISSUE TITLE
Pkg's gc failing on  julia 1.7.0

### ISSUE THREAD

**Issue by BeastyBlacksmith** [JuliaLang/julia#43286]
The error message is
```julia
[ Info: We haven't cleaned this depot up for a bit, running Pkg.gc()...
┌ Error: GC failed
│   exception =
│    TOML Parser error:
│    ~/.julia/logs/manifest_usage.toml:1103:0 error: invalid bare key character: '\n'
│      3Z
│        ^
└ @ Pkg /buildworker/worker/package_linux64/build/usr/share/julia/stdlib/v1.7/Pkg/src/Pkg.jl:644
```

**Comment by BeastyBlacksmith** [comment]
There is a suspiciously looking `3Z` on this line in the file
```
[["~/.julia/dev/OCTOPOS/web/Manifest.toml"]]
time = 2021-11-10T11:07:35.083Z
3Z
```
But there are also more lines like this in the file

**Comment by Seelengrab** [comment]
The line with `time = ` is a valid ISO-8601 datetime string with timezone information (that's the `Z`, here for Zulu or UTC time) - what shouldn't be there is imo the lonely `3Z`. Does this occur at the end somewhere? It looks like a reused buffer with different lengths didn't get properly truncated :thinking:

**Comment by giordano** [comment]
Yeah, in the Yggdrasil buildmachine we have _plenty_ of those lines.  I suspect that may be due to parallel access to the file.

**Comment by jonniedie** [comment]
Whats the best way to handle this if it happens to me? Just delete that file and let it rebuild?

**Comment by giordano** [comment]
In Yggdrasil that file is close to 1 million lines, so for me it's easier to delete it than trying to fix it (something which I actually tried once, but gave up after half an hour when I realised I was at 10% of the file)
